frontend/src/components/header.tsx

```
@@ -0,0 +1,39 @@
+        <section className="links_container">
+          <span className="link">
+            <Link to="/table">Table</Link>
+          </span>
+          <span className="link">
+            <Link to="/graph">Graph</Link>
+          </span>
+        </section>
+      </section>
+
+      <span className="reset">
+        <button onClick={reset_to_main}>reset</button>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)